README.md

```diff
@@ -64,7 +64,8 @@ Although BlockHound supports [the SPI mechanism to integrate with](https://githu
 1. [Project Reactor](https://projectreactor.io)  
   Version 3.2.x is supported out of the box.  
   Starting with `reactor-core` version 3.3.0, there is [a built-in integration in Reactor itself](https://github.com/reactor/reactor-core/blob/v3.3.0.RELEASE/reactor-core/src/main/java/reactor/core/scheduler/ReactorBlockHoundIntegration.java) that uses [the SPI](https://github.com/reactor/BlockHound/blob/master/docs/custom_integrations.md).
-2. [RxJava 2](https://github.com/ReactiveX/RxJava/)
+2. [RxJava 2](https://github.com/ReactiveX/RxJava/) is supported.  
+    RxJava 3 and further versions of RxJava will require an SPI to be implemented, either by the framework or user. See [this PR to RxJava](https://github.com/ReactiveX/RxJava/pull/6692) with an example of the SPI's implementation.
```

Comment:
I think this is a little misleading and implies RxJava itself needs to implement an SPI. Blockhound could offer an extension artifact for rx3 too

Reply:
BlockHound team has no intention of implementing nor maintaining the extension for RxJava3. One of the reasons is a need to work with internal implementation details of the framework that may change.
Even Reactor's integration leaves in reactor-core, not BlockHound.

If Netty, Kotlin Coroutines and a few more projects were able to implement the SPI, I see no reason why RxJava should not do the same.

Reply:
You're still implying in the wording here that it requires rxjava to implement something in order for it to work, which isn't true. Not every project is willing to add dependencies, rxjava is historically focused on having zero dependencies.

Reply:
>  rxjava is historically focused on having zero dependencies.

As well as Netty.
But, implementing SPI does not require having BlockHound as a dependency - it can be optional or compileOnly.

Also, anyone can integrate BlockHound with RxJava in their project, as per the docs:
https://github.com/reactor/BlockHound/blob/master/docs/custom_integrations.md

Reply:
Yes. My point is that the line I commented on here is ambiguous and implies that RxJava requires a change to be implemented first

Reply:
It doesn't. It explicitly states "an SPI to be implemented, either by the framework or user". It does not say that RxJava needs to do anything on its side to make the detection work.

It would be better if they accepted my PR, so that BlockHound is supported OOTB by RxJava (similar to Kotlin, Netty and other libraries that integrated with BlockHound), but, if the maintainers of RxJava don't want to, it is up to their users to integrate, either in their projects or by working on a 3rd party library that with glue both.